lua/net.lua

```diff
@@ -12,14 +12,16 @@ return function (gatekeeper_server)
 	local front_ports = {"enp133s0f0"}
 	local front_ips  = {"10.0.1.1/24", "2001:db8:1::1/48"}
 	local front_bonding_mode = staticlib.c.BONDING_MODE_ROUND_ROBIN
-	local front_vlan_tag = 0x123
+	local front_ipv4_vlan_tag = 0x123
+	local front_ipv6_vlan_tag = 0x123
```

Comment:
Replace line `local front_ipv6_vlan_tag = 0x123` with `local front_ipv6_vlan_tag = front_ipv4_vlan_tag`.

Do the same for the back interface as well.